content/doc/book/blueocean/index.adoc

```diff
@@ -53,51 +76,42 @@ allowfullscreen></iframe>
 endif::[]
 
 
-== Join the community
+== Frequently asked questions
 
-There a few ways you can join the community:
-
-. Chat with the community and development team on Gitter image:https://badges.gitter.im/jenkinsci/blueocean-plugin.svg[link="https://gitter.im/jenkinsci/blueocean-plugin?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge"]
-. Request features or report bugs against the link:https://issues.jenkins-ci.org/[`blueocean-plugin` component in JIRA].
-. Subscribe and ask questions on the link:https://groups.google.com/forum/#!forum/jenkinsci-users[Jenkins Users mailing list].
-. Developer? We've link:https://issues.jenkins-ci.org/issues/?filter=16142[labeled a few issues] that are great for anyone wanting to get started developing Blue Ocean. Don't forget to drop by the Gitter chat and introduce yourself!
-
-
-== Frequently Asked Questions
 
 === Why does Blue Ocean exist?
 
 The world has moved on from developer tools that are purely functional to
-developer tools being part of a "developer experience". That is to say, it's no
+developer tools being part of a "developer experience". That is to say, it is no
 longer about a single tool but the many tools developers use throughout the day
-and how they work together to achieve a workflow that's beneficial for the
-developer - this is Developer Experience.
+and how they work together to achieve a workflow that is beneficial for the
+developer - this is "developer experience".
 
 Developer tools companies like Heroku, Atlassian and Github have raised the bar
 for what is considered good developer experience, and developers are
-increasingly expecting exceptional design. In recent years developers are
-becoming more rapidly attracted to tools that are not only functional but are
+increasingly expecting exceptional design. In recent years, developers have
+become more rapidly attracted to tools that are not only functional but are
 designed to fit into their workflow seamlessly and are a joy to use. This shift
 represents a higher standard of design and user experience that Jenkins needs
-to rise to meet.
+to rise and meet.
```

Comment:
rise to meet - as in 
"This shift represents a higher standard of design and user experience.  Jenkins needs to rise to meet this higher standard."

Reply:
Ah - sorry - I'll revert.